File: 250408/가장 짧은 문자열/shortest-string.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String str1 = sc.next();
        String str2 = sc.next();
        String str3 = sc.next();

        int str1_len = str1.length();
        int str2_len = str2.length();
        int str3_len = str3.length();

        int max = Integer.MIN_VALUE;
        if(str1_len > max) {
            max = str1_len;
        }
        if(str2_len > max) {
            max = str2_len;
        }
        if(str3_len > max) {
            max = str3_len;
        }
        int min = Integer.MAX_VALUE;
        if(str1_len < min) {
            min = str1_len;
        }
        if(str2_len < min) {
            min = str2_len;
        }
        if(str3_len < min) {
            min = str3_len;
        }

        System.out.println(max - min);

        sc.close();
    }
}

File: 250408/가장 짧은 문자열/shortest-string.cpp
#include <iostream>
#include <string>
#include <climits>
using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);

    string str1, str2, str3;
    cin >> str1 >> str2 >> str3;
    int length1 = str1.length();
    int length2 = str2.length();
    int length3 = str3.length();
    int max = INT_MIN;
    int min = INT_MAX;
    if(length1 > max)
        max = length1;
    if(length2 > max)
        max = length2;
    if(length3 > max)
        max = length3;
    if(length1 < min)
        min = length1;  
    if(length2 < min)
        min = length2;
    if(length3 < min)
        min = length3;

    cout << max - min;
    return 0;
}